downloads-generation/cross_validation_class1/README.md

```diff
@@ -0,0 +1,7 @@
+# Cross validation of standard Class I models
+
+This download contains cross validation results and intermediate data for
```

Comment:
Possibly topic for future brainstorming: can we use a better noun than "download"? It's so often a verb that reading this awkward.

Reply:
I'd be fine to change it but I didn't have much luck thinking of a better name for something that includes data, models, cross validation results, etc. Any idea on a better term?